src/rpcblockchain.cpp

```diff
@@ -605,6 +605,7 @@ bool CheckMessageSignature(std::string sAction,std::string messagetype, std::str
      if (sAction=="D" && messagetype=="beacon") strMasterPubKey = msMasterProjectPublicKey;
 	 if (sAction=="D" && messagetype=="poll")   strMasterPubKey = msMasterProjectPublicKey;
 	 if (sAction=="D" && messagetype=="vote")   strMasterPubKey = msMasterProjectPublicKey;
+     if (messagetype=="constblkreward")         strMasterPubKey = msMasterProjectPublicKey;
```

Comment:
Before I forget, since this is in MemorizeMessage the reward constant will only be valid for 6 months. We might want to either extend that scan or refresh it biannually.

Reply:
I think it should be ultimately locked at a set value.